ui/packages/catalog-ui-search/src/main/webapp/react-component/container/workspace-item-container/workspace-item-container.tsx

```diff
@@ -30,7 +30,10 @@ interface State {
 const getStateFromWorkspace = (workspace: any) => {
   return {
     title: workspace.get('title'),
-    date: moment(workspace.get('metacard.modified')).fromNow(),
+    date: moment(
+      workspace.get('metacard.modified'),
```

Comment:
Are these attributes for workspaces not coming in as ISO8601?  I think this may be unnecessary with the merging of https://github.com/codice/ddf/pull/4197